cmd/qvain-backend/api_helpers.go

```diff
@@ -146,50 +147,49 @@ func apiVersion(w http.ResponseWriter, r *http.Request) {
 
 // dbError handles database errors. It returns more specific API messages for predefined errors
 // that might be relevant for the user. Other errors return `database error` with a 500 status code.
-func dbError(w http.ResponseWriter, err error) bool {
+// Also logs error message to backend terminal
+func dbError(w http.ResponseWriter, err error, logger *zerolog.Logger) *zerolog.Event {
 	switch err {
 	case nil:
-		return false
+		return nil
```

Comment:
why this change?

Reply:
we are returning object instead of bool (which it was earlier)